pandas/tests/dtypes/test_inference.py

```diff
@@ -568,6 +568,43 @@ def test_maybe_convert_objects_nullable_integer(self, exp):
 
         tm.assert_extension_array_equal(result, exp)
 
+    def test_maybe_convert_objects_bool_nan(self):
+        # GH32146
+        arr = pd.Index([True, False, np.nan], dtype=object)
+        exp = np.array([True, False, np.nan], dtype=object)
+        out = lib.maybe_convert_objects(arr.values, safe=1)
+        tm.assert_numpy_array_equal(out, exp)
+
+    @pytest.mark.parametrize(
+        "arr, dropna, exp",
+        [
+            (
+                pd.Series([False, True, True, pd.NA]),
+                False,
+                pd.Series([2, 1, 1], index=[True, False, pd.NA]),
+            ),
+            (
+                pd.Series([False, True, True, pd.NA]),
+                True,
+                pd.Series([2, 1], index=[True, False]),
+            ),
+            (
+                pd.Series(range(3), index=[True, False, np.nan]).index,
+                False,
+                pd.Series([1, 1, 1], index=[True, False, pd.NA]),
+            ),
+        ],
+    )
+    def test_maybe_convert_objects_value_counts(self, arr, dropna, exp):
```

Comment:
im not sure exactly where value_counts tests live, but pretty sure it isnt here

Reply:
Moved it to tests.series.methods.test_value_counts.py